### PR TITLE
UI Test:Initiate e2e UI tests

### PR DESCRIPTION
**Pull request by BathiyaL**
Test scripts/functions verified on APIM v4.1.0

### CODE REVIEW

File: tests/cypress/integration/e2e/developerFundamentalScenarios/00-defining -users-and-roles.spec.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+/// <reference types="cypress"/>
```
Comment (madushadhanushka) [inline tests/cypress/integration/e2e/developerFundamentalScenarios/00-defining -users-and-roles.spec.js:1]: Shall we remove spaces in the file

File: tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "apiCreatorUserRole": "apicreator",
+  "apiPublisherUserRole": "apipublisher",
+  "newUserCreator": "creator",
+  "newUserPublisher": "publisher",
+  "newUserSubscriber": "subscriber",
+  "newUserMike": "mike",
+  "newUserPassword": "password"
+}
+
+
```
Comment (madushadhanushka) [inline tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json:11]: Lets remove this additional empty lines

Comment (BathiyaL) [reply tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json:11]: sure, shall I update this in the next PR?

File: tests/cypress/integration/e2e/developerFundamentalScenarios/00-defining -users-and-roles.spec.js (hunk 1 above)
Comment (BathiyaL) [reply tests/cypress/integration/e2e/developerFundamentalScenarios/00-defining -users-and-roles.spec.js]: Fixed

File: tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json (hunk 2 above)
Comment (BathiyaL) [reply tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json]: Fixed

**Review by chanaka3d — APPROVED** [review]

**Pull request merged**